### PR TITLE
[ALE-08] Add 'add/remove' favorite feature in home page

### PR DESCRIPTION
**Pull request by alejandrobenjumeaGL**
Feb-25-2020 14-33-20

### CODE REVIEW

File: src/api/index.js [hunk 1]
```diff
@@ -133,42 +130,52 @@ class Api {
...
+    )();
+  }
+
   /**
    * Add favorite
    * @param  {} track - track entity
    * @return {number[]} Favorites list
    */
-  addFavorite(track) {
+  addFavorite(track, n) {
```
Comment (julianfigueroaGL) [inline src/api/index.js:152]: please add the JSDoc description for the **n** parameter

File: src/api/index.js [hunk 2]
```diff
@@ -133,42 +130,52 @@ class Api {
...
+    return this.getFavoritesTracks(n);
   }
 
   /**
    * Remove favorites id
    * @param  {} id - Id to remove
    * @return {Track[]} Favorites list
    */
-  removeFavorite(id) {
+  removeFavorite(id, n) {
```
Comment (julianfigueroaGL) [inline src/api/index.js:184]: SAB

File: src/views/home/script.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import ListPodcast from '@/components/list-podcast/list-podcast.vue';
+import api from '@/api';
+import { getRandomPodcastCategory } from '@/utils';
+import { DEFAULT_PODCAST_FILTER_PARAMS } from '@/utils/constants';
+
+import { assocPath, map, take } from 'ramda';
+
+const maxItemns = 5;
```
Comment (julianfigueroaGL) [inline src/views/home/script.js]: Please rename this var as _maxItems_

File: src/views/home/script.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+      },
+      deep: true,
+    },
+  },
+  components: {
+    ListPodcast,
+  },
+  async created() {
+    this.podcasts = await api.search(getRandomPodcastCategory(), this.getSearchParams());
+    this.favorites = take(5, api.getFavoritesTracks());
```
Comment (julianfigueroaGL) [inline src/views/home/script.js]: instead of the value 5, could we use the maxItems var?

Comment (alejandrobenjumeaGL) [reply src/views/home/script.js]: done

File: src/views/home/script.js (hunk 3 above)
Comment (alejandrobenjumeaGL) [reply src/views/home/script.js]: done

File: src/api/index.js (hunk 1 above)
Comment (alejandrobenjumeaGL) [reply src/api/index.js:152]: done

File: src/api/index.js (hunk 2 above)
Comment (alejandrobenjumeaGL) [reply src/api/index.js:184]: done

**Review by julianfigueroaGL — APPROVED** [review]
Well done

**Pull request merged**